src/custom-element.js

```diff
@@ -65,6 +65,9 @@ const UpgradeState = {
  */
 let templateTagSupported;
 
+/** @type {!WeakMap<!./service/ampdoc-impl.AmpDoc, boolean>} */
+const docInitializedMap = new WeakMap();
```

Comment:
Nit: I'm not sure this is `custom-elements.js`'s responsibility. What if we moved the map over be inside of `stubElementsForDoc` and just call the cb in `createCustomElementClass` unconditionally?

Reply:
Yeah. I see your point. I tried to refactor it a bit. PTAL.